gekko.js

```diff
@@ -59,3 +59,7 @@ pipeline({
   mode: mode
 });
 
+process.on('unhandledRejection', (reason, p) => {
+  console.log(`Unhandled Rejection at: ${p}, Reason: ${reason}`);
```

Comment:
can we use the the logger that comes with Gekko, and use the `error` method to log to stdout?

That logger can be found here: https://github.com/askmike/gekko/blob/develop/core/log.js

Reply:
Yep, makes sense.

Reply:
Updated and pushed changes.